### PR TITLE
[gh-pages][reduced-operator-config-file.md] Add documentation about globally allowed types.

### PR DESCRIPTION
**Pull request by edgchen1**
### Description

Add documentation about globally allowed types.

https://edgchen1.github.io/onnxruntime/docs/reference/operators/reduced-operator-config-file.html

### Motivation and Context

Document additional configuration file syntax.

### CODE REVIEW

**Review by natke — APPROVED** [review]

File: docs/build/eps.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ See more information on the TensorRT Execution Provider [here](../execution-prov
    * The path to the CUDA installation must be provided via the CUDA_PATH environment variable, or the `--cuda_home` parameter. The CUDA path should contain `bin`, `include` and `lib` directories.
    * The path to the CUDA `bin` directory must be added to the PATH environment variable so that `nvcc` is found.
    * The path to the cuDNN installation (path to cudnn bin/include/lib) must be provided via the cuDNN_PATH environment variable, or `--cudnn_home` parameter.
-     * On Windows, cuDNN requires [zlibwapi.dll](https://docs.nvidia.com/deeplearning/cudnn/install-guide/index.html#install-zlib-windows). Feel free to place this dll under `path_to_cudnn/bin`  
+     * On Windows, cuDNN requires [zlibwapi.dll](https://docs.nvidia.com/deeplearning/cudnn/archives/cudnn-850/install-guide/index.html#install-zlib-windows). Feel free to place this dll under `path_to_cudnn/bin`
```
Comment (skottmckay) [inline docs/build/eps.md:107]: This is a link to cuDNN 8.5 doco, but the TRT section says it's built with cuDNN 8.9.

Should we link to the 8.9 documentation as an example, as well as the overall archive (https://docs.nvidia.com/deeplearning/cudnn/archives/index.html) and tell the user to check the Installation Guide fir the specific release they're using as the install instructions frequently change (e.g. it goes back and forth between putting the cudnn dlls in the cuda install path and putting them in a separate directory)

It also feels like the wrong place as cuDNN is used by the CUDA EP and not just TRT so these instructions apply to both.

Comment (edgchen1) [reply docs/build/eps.md]: I reverted this change. Not sure how best to handle it now and clean up version-specific instructions.

FWIW, cuDNN 8.9.7 install instructions don't have the Windows zlib section anymore:
https://docs.nvidia.com/deeplearning/cudnn/archives/cudnn-897/install-guide/index.html#install-zlib-windows

Comment (edgchen1) [reply docs/build/eps.md]: https://github.com/microsoft/onnxruntime/issues/19473

**Review by skottmckay — APPROVED** [review]

File: docs/build/eps.md (hunk 1 above)
Comment (skottmckay) [reply docs/build/eps.md]: The zlib install has often been a step people miss, so if they're directly including the necessary code that _might_ reduce the 'CUDA dll doesn't load' GH issues.

**Review by natke — APPROVED** [review]

**Pull request merged**